src/shared/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Alert, AlertTitle, Box } from "@mui/material";
+import { useSelector } from "react-redux";
+import { SerializedError } from "@reduxjs/toolkit";
+import { RootState } from "../../store";
+
+export const ErrorMessage = () => {
+
+    const error = useSelector<RootState, null | SerializedError>(state => state.cardsSlice.error);
```

Comment:
This is a shared component but you select error from cardSlice.
So error should be passed in props